Kitodo/src/main/resources/messages/errors_en.properties

```diff
@@ -86,18 +86,18 @@ metadataDigitalDocumentError=DigitalDocument error at\: {0}
 metadataEditorThreadLock=Another file gets processed in the background. Please try again after some seconds.
 metadataError=Critical error within the metadata file with a possible loss of data. Please inform the administrator.
 metadataFileNotFound=Metadata file not found \:
-metadataInvalidData=invalid input
+metadataInvalidData=The input "{0}" in field {1} is invalid.
 metadataIsEmpty=contains no data
-metadataMandatoryElement=missing input for\:
-metadataMissingIdentifier=missing value for identifier
+metadataMandatoryElement=Missing input for {0}.
+metadataMediaError=A structure has media assigned which aren't declared.
+metadataMediaUnassigned=The media unit is not assigned to a structure\:
+metadataMissingIdentifier=Missing value for the identifier.
 metadataNotAllowed={0} {1} is not allowed in new element {1}.
-metadataNotEnoughElements=have to exist at least once
```

Comment:
The following error messages seems to be incomplete!?

Reply:
Exactly. That's why I changed it.